src/ray/gcs/gcs_client/test/global_state_accessor_test.cc

```diff
@@ -272,10 +304,13 @@ TEST_F(GlobalStateAccessorTest, TestWorkerTable) {
 }
 
 // TODO(sang): Add tests after adding asyncAdd
-TEST_F(GlobalStateAccessorTest, TestPlacementGroupTable) {
+TEST_P(GlobalStateAccessorTest, TestPlacementGroupTable) {
   ASSERT_EQ(global_state_->GetAllPlacementGroupInfo().size(), 0);
 }
 
+INSTANTIATE_TEST_CASE_P(RedisRemovalTest, GlobalStateAccessorTest,
```

Comment:
Build error: `INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Werror,-Wdeprecated-declarations]`
```suggestion
INSTANTIATE_TEST_SUITE_P(RedisRemovalTest, GlobalStateAccessorTest,
```